VirtoCommerce.Domain/Catalog/Services/IProductAssociationSearchService.cs

```diff
@@ -0,0 +1,10 @@
+using VirtoCommerce.Domain.Catalog.Model;
+using VirtoCommerce.Domain.Catalog.Model.Search;
+
+namespace VirtoCommerce.Domain.Catalog.Services
+{
+    public interface IProductAssociationSearchService
+    {
+        SearchResult SearchProductAssociations(ProductAssociationSearchCriteria criteria);
```

Comment:
Instead `SearchResult` should be the `GenericSearchResult<CatalogProduct>` type

Reply:
understand, thank you, I'll replace type